src/modules/requests/requests.repository.ts

```diff
@@ -37,24 +37,24 @@ export class RequestsRepository {
     };
   }
 
-  private setSearch(findAll: FindAll) {
+  private setSearch(findAll: FindAll, filter: Object) {
     const search: Array<Record<string, any>> = [];
     const keyword = findAll.q;
 
-    if (findAll.q) {
-      search.push({ username: Like(`%${keyword}%`) });
-      search.push({ phone_number: Like(`%${keyword}%`) });
-    }
+    const byUsername = { ...filter, username: Like(`%${keyword}%`) };
```

Comment:
```suggestion
    const likeByUsername = { ...filter, username: Like(`%${keyword}%`) };
```

```suggestion
    const whereByUsername = { ...filter, username: Like(`%${keyword}%`) };
```

Mungkin naming nya bisa di ubah kang? Bagusnya di ganti apa yah

Reply:
mungkin `matchWithUsername` ?

Reply:
> mungkin `matchWithUsername` ?

Boleh kang, yang penting bukan kata sambung di depannya 😁